packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.js

```diff
@@ -15,15 +15,21 @@ import {
 export default class PageTabs extends React.Component {
   render() {
     const { tabs, slug } = this.props;
-    const currentTab = slug.split('/').filter(Boolean).slice(-1)[0];
+    const newSlug = slug.replace(/[’']/g, ''); // removes any apostrophes from slug
+    const currentTab = newSlug.split('/').filter(Boolean).slice(-1)[0];
 
     const pageTabs = tabs.map((tab) => {
-      const slugifiedTab = slugify(tab, { lower: true });
+      const tempTab = tab.replace(/[’']/g, ''); // removes any apostrophes from tab
+      const slugifiedTab = slugify(tempTab, { lower: true });
       const selected = slugifiedTab === currentTab;
-      const currentTabRegex = new RegExp(`${currentTab}(?!-)`);
+      const currentTabRegex = new RegExp(`(?<!-)${currentTab}(?!-)`);
```

Comment:
While this works for `/conceptual-model/model` it wouldn't solve the issue for something like `/navigation/nav`. We'd still end up replacing the first instance of `nav` and get broken urls.

What if instead, we use a `$` to indicate the end of the string (while allowing for an optional trailing slash)?

```suggestion
      const currentTabRegex = new RegExp(`${currentTab}/?$`);
```

Reply:
Good point @vpicone. Would it still work if the url is pointing to a specific section of the page and the tab isn't necessarily the end of the string? i.e. `/conceptual-model/model/#section-of-page`

Reply:
@jnm2377 good question, hash links don't work if there's a slash in between though ( so it'd be `/conceptual-model/model#section-of-page` and `/conceptual-model/model#section-of-page/`) I'm not even sure they work with a trailing slash.

Let me know if it doesn't work with section links and we can brainstorm

Reply:
this example I was just looking at has a slash in between: https://www.ibm.com/design/language/iconography/pictograms/contribute/#export-svgs-from-adobe-illustrator

Reply:
@jnm2377 oh wow sure enough maybe this is a server setting? Good catch. We can get around this by looking for _either_ a hashtag _or_ the end of the line. I'll edit my suggestion.

Reply:
@jnm2377 updated to add an optional group for hash links

Reply:
Seems to be working. I added another testing example under `Guides > Navigation > Nav`